packages/api/migrations/20200820005843-create-user.js

```diff
@@ -16,10 +16,6 @@ module.exports = {
         allowNull: false,
         type: Sequelize.STRING,
       },
-      password: {
-        allowNull: false,
-        type: Sequelize.STRING,
-      },
```

Comment:
Modifying existing migrations that have already been merged is a headache waiting to happen. The migration tooling tracks which ones have already been applied and skips those, which means in production/staging, this change won't be made. Thus, your testing schema will be different from what's in production, which likely means bugs.

To effectively alter the schema, you'll need to create a new migration.